var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -289,13 +290,24 @@ class Llvm(CMakePackage, CudaPackage):
     depends_on("zstd build_system=cmake", when="+zstd")
 
     # lldb dependencies
-    with when("+lldb +python"):
-        depends_on("swig")
+    with when("+lldb"):
+        depends_on("libedit")
+        depends_on("libxml2")
+        depends_on("lua@5.3", when="+lua")  # purposefully not a range
```

Comment:
@trws said that Lua is an alternative to Python for scripting. So, we should also be making it a "run" type dependency.

Reply:
it's embedded, linking to liblua is sufficient

Reply:
Is it insufficient for using lua 5.4 or later here?